src/iris/ui/templates/plan.html

```diff
@@ -371,6 +373,13 @@ <h4><i class="icon-plus icon">+</i> Add application tracking template</h4>
         {{/each}}
       </select>
     </p>
+    <p>
+      <label class="item-label light" data-toggle="tooltip" title="If set to true, this notification won't generate an alert if it fails to find a valid target.">Optional</label>
+      <select class="content-template form-control border-bottom" data-type="optional">
+          <option value="0">False</option>
```

Comment:
When viewing a plan, will this dropdown have the proper option selected?

Reply:
Yes, the default option selected is False.

Reply:
Why not Yes/No or Enabled/Disabled or On/Off instead of True/False? Exposing boolean names feels like an implementation detail that the user might not care about.

Reply:
There wasn't a specific reason for it to be worded as True/False but you bring up a good point, I think Yes/No might in fact be a bit clearer for the user in this case.